From b66d24b45547a96094ea9e246f6ad5068dc37e5a Mon Sep 17 00:00:00 2001
From: Carlos Rafael Giani <dv@pseudoterminal.org>
Date: Wed, 28 May 2014 12:29:57 +0200
Subject: [PATCH] Enable share group workaround for Vivante GPUs

This fixes the "blue background" bug with transparent HTML canvas areas

Upstream-Status: Inappropriate [other]
This fix enables the workarounds for Vivante GPUs on all platforms, even on
ones where it may not be broken; even limiting it to android and linux might
not be enough

Signed-off-by: Carlos Rafael Giani <dv@pseudoterminal.org>
---
 gpu/config/gpu_driver_bug_list_json.cc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/gpu/config/gpu_driver_bug_list_json.cc b/gpu/config/gpu_driver_bug_list_json.cc
index 4db401e..35d5a4f 100644
--- a/gpu/config/gpu_driver_bug_list_json.cc
+++ b/gpu/config/gpu_driver_bug_list_json.cc
@@ -472,9 +472,6 @@ const char kGpuDriverBugListJson[] = LONG_STRING_CONST(
       "id": 34,
       "cr_bugs": [179250, 229643, 230896],
       "description": "Share groups are not working on (older?) Vivante drivers",
-      "os": {
-        "type": "android"
-      },
       "gl_extensions": {
         "op": "contains",
         "value": "GL_VIV_shader_binary"
-- 
1.8.3.2

